<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJabatan</class>
 <widget class="QWidget" name="FormJabatan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="kd_jabatan"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nama_jabatan"/>
    </item>
    <item>
     <widget class="QLineEdit" name="tunjangan"/>
    </item>
    <item>
     <widget class="QLineEdit" name="gaji_pokok"/>
    </item>
    <item>
     <widget class="QLineEdit" name="deskripsi"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>383</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnTambah">
      <property name="text">
       <string>Tambah</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBersih">
      <property name="text">
       <string>BERSIH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>160</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kode Jabatan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nama Jabatan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tunjangan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gaji Pokok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deskripsi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>291</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>JABATAN PERUSAHAAN</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableJabatan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>kd_jabatan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama_jabatan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tunjukan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gaji_pokok</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>deskripsi</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editcari">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>229</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>158</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cari Jabatan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
